### DISCUSSION TITLE
C3 Festival Ticket Raffle

### DISCUSSION THREAD

**Discussion by Mokkapps** [Mokkapps/mokkapps-de-discussions#5] [General]
# Win one of three free tickets for the C3 Dev Festival

I got the opportunity to raffle 3 in-person tickets to first edition of the C3 Dev Conference. Just comment in this discussion and I will select the three winners at 23th April 2024.

## C3 Dev Festival - the contemporary software engineering and design festival

![Image 1](https://media.graphassets.com/tqmn3lyR4OIdVBgJ7Avg)

### What is it about? 
Check out the 🎨C3, a contemporary software engineering & design festival! 

📚Code, career, creativity
🎮12 artists & 30 speakers
💡Speed Networking
& many more

Amsterdam, June 14-15, 2024 → https://c3fest.com

**Comment by simonwuyts** [comment]
Looks very inspiring, I would love to go there!

**Comment by leonardigabriele** [comment]
It looks amazing! 🙏

**Comment by MethodJules** [comment]
Looks really interesting. I would love to go to this event.

**Comment by Lonzetorm** [comment]
What a wonderful opportunity! Would like to participate

**Comment by atmoner** [comment]
Ooh! Seem really cool!!

**Comment by Mokkapps** [comment]
The winners are @atmoner, @simonwuyts  and @MethodJules. Please send me a mail to [mail@mokkapps.de](mailto:mail@mokkapps.de) to get your free ticket.

🥳 Congratulations!
